Codeinwp/neve
# feat: added control to list font pairs inside typography section

### Summary

Added a new section under Typography called `Font pairings`.
Added a new control for that section that holds the available Font pairings, and allows the user to change the Heading and body font at the same time.

A new filter was added `neve_font_pairings`. It can be used to change the font pairings listed.

The font pair array allows for setting preview sizes for the font that are being used when displaying the font inside the control. This helps us have control over how the fonts are being displayed and avoid issues with fonts going out of bounds.

### Will affect visual aspect of the product

YES

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/211565556-fcae0c83-093f-47fa-a538-b7a1f35bac40.png)

![image](https://user-images.githubusercontent.com/23024731/211565647-22a751c5-dbbb-4d80-a050-d4a99e93b1a8.png)

![image](https://user-images.githubusercontent.com/23024731/211565722-b061a6fc-578e-43a7-98c9-01c22e2eba69.png)

### Test instructions

1. Check that the fonts update correctly for heading and body when a pair is selected.
2. Check that they display correctly on the front-end and inside the Customizer.
3. Check that when global colors are changed the preview for the control changes. E.g. the colors used to display the preview inside the control are: `Dark Background`, `Text Dark Background` for hover and `Site Background`, `Text Color` for normal state.
4. Check the values are reset to the Default value of the typography Control when `Reset to customizer defaults`
 is clicked.

Closes Codeinwp/neve-pro-addon#2190.
